pkg/controller/util/node/controller_utils.go

```diff
@@ -430,36 +357,23 @@ func GetTenantPartitionManagersFromKubeClients(clients []clientset.Interface, st
 }
 
 func GetTenantPartitionManagersFromServerNames(tenantServers []string, stop <-chan struct{}) ([]*TenantPartitionManager, error) {
```

Comment:
This function now takes kubeconfigs instead of server ips. Please rename correspondingly

Reply:
will fix. somehow this is not shown when i did the global search in the previous committed changes.